docs/dev/README.md

```diff
@@ -79,3 +79,11 @@ realize your changes.
     ```bash
     > cd app && npm run build
     ```
+
+## Tests
+
+Please confirm the changes don't break anything running the test suite. You need to [install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-kubectl) before.
+
+```bash
+> npm test
+```
```

Comment:
great, thanks for adding this. i think @myan9 has been successful also with the Kubernetes support that Docker offers directly (e.g. on mac, you can enable this from the Docker menu.

Reply:
Right, on Kubernetes official document, there're 3 local-machine solutions to easily get started with Kubernetes. We could infer developers about this: https://kubernetes.io/docs/setup/pick-right-solution/#local-machine-solutions

Reply:
I think Minikube is easier for developers, but we can add both ways to the documentation if you prefer.